### PR TITLE
allows externally defined ssl cert

### PR DESCRIPTION
**Pull request by whereismyjetpack**
Signed-off-by: Dann Bohn <dannbohn@gmail.com>

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [ ] Chart Version bumped
- [ ] Variables are documented in the README.md

### CODE REVIEW

**Comment by whereismyjetpack** [comment]
/assign @nmengin

@dtomcej new PR check it out. 

Thanks for hanging in there with me.

File: stable/traefik/values.yaml
Comment (dtomcej) [inline stable/traefik/values.yaml]: Can this be commented out by default?

**Review by dtomcej — CHANGES_REQUESTED** [review]
Looks good. Just one small change request!

File: stable/traefik/values.yaml
Comment (whereismyjetpack) [reply stable/traefik/values.yaml]: I commented this out and tested locally. check it out!

**Comment by Art3mK** [comment]
doh, tired waiting for this to be merged. Should I open my own PR for that?

ping @whereismyjetpack, could you, please, bump chart version?

**Comment by whereismyjetpack** [comment]
sorry for the delay. chart version bumped.

**Comment by whereismyjetpack** [comment]
@dtomcej can you check this out? The UI is telling me there's a merge conflict, I don't have write access to fix it, so i pushed up the offending file and the UI did not update. please let me know if there's something else i need to do

Thanks!

**Comment by dtomcej** [comment]
Hi @whereismyjetpack, I don't have access to push to your branch either.

A rebase may be required to fix the merge conflict.

**Pull request closed without merging**
